server/index.js

```diff
@@ -102,6 +102,18 @@ app.put('/qa/answers/:answerId/report', (req, res) => {
     });
 });
 
+app.get('/qa/questions/:id', (req, res) => {
```

Comment:
I also implemented this slightly different. Instead of id at the end, i did:

```
app.get('/qa/questions', (req, res) => {
  const productId = req.query.product_id;
```
Then in my client api request, I pass product_id as a query parameter like:
```
axios.get(`qa/questions?product_id=${productId}`)
 ```
Just how I implemented, I can work with either way.